rocAL/source/decoders/image/decoder_factory.cpp

```diff
@@ -47,6 +48,9 @@ std::shared_ptr<Decoder> create_decoder(DecoderConfig config) {
         case DecoderType::ROCJPEG_DEC:
             return std::make_shared<HWRocJpegDecoder>(config.get_hip_stream());
             break;
+        case DecoderType::FUSED_CROP_ROCJPEG_DEC:
+            return std::make_shared<FusedCropRocJpegDecoder>();
```

Comment:
Why doesn't FusedCropRocJpegDecoder require HIP stream in the constructor when rocJpegDecoder requires it?